app/models/preference.rb

```diff
@@ -17,6 +17,8 @@
 #  index_preferences_on_user_id  (user_id)
 #
 class Preference < ApplicationRecord
+  Preference::MAX_PREFERENCES = 5
```

Comment:
@enzofab91 I added this again since it is required by the index, and I want to test that after updating the preference, the success notice is shown by the index